src/components/organisms/AdminVenueView/AdminVenueView.scss

```
@@ -0,0 +1,52 @@
+@import "scss/constants.scss";
+
+.AdminVenueView {
+  display: flex;
+  justify-content: space-around;
+  padding: 20px 0;
+
+  @media (max-width: 1024px) {
+    align-items: center;
+  }
+
+  @media (max-width: 800px) {
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  &__options {
+    display: flex;
+    width: 80%;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__tab {
+    position: relative;
+    display: flex;
+    flex-grow: 1;
+    text-align: center;
+    align-self: center;
+    cursor: pointer;
+    color: rgba($white, 0.6);
+    text-decoration: none;
+
+    @media (max-width: 1024px) {
+      width: 50%;
+    }
+
+    @media (max-width: 800px) {
+      width: 100%;
+    }
+
+    &:hover {
+      color: $white;
+    }
+
+    &.selected {
```

Can we make this a BEM modifier rather than a separate class?

```suggestion
    &--selected {
```